ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Update.scala

```diff
@@ -59,12 +59,16 @@ object Update {
     * @param recordTime
     *   The ledger-provided timestamp at which the party was allocated.
     *
+    * @param submissionId
+    *   The submissionId of the command which requested party to be added.
+    *
     */
   final case class PartyAddedToParticipant(
       party: Party,
       displayName: String,
       participantId: ParticipantId,
-      recordTime: Timestamp)
+      recordTime: Timestamp,
+      submissionId: String)
```

Comment:
We have different wrapped types for the IDs instead of just using `String` directly. See `ApplicationId`, `CommandId`, `PackageId`, etc...

Reply:
@gerolf-da do we have any type for submissionId?  this is kinda like a CommandId but isn't there a typical format convention for CommandId, or is it just an opaque string?  typing submissionId as commandId may be a bit misleading though

Reply:
You can introduce a `SubmissionId` the same way as `CommandId`. And yes, they are just opaque strings (adhering to the `LedgerString` restrictions).

Reply:
Use `SubmissionId`?

Reply:
addressed